R/config.R

```diff
@@ -1,10 +1,12 @@
 gsconfig <- new.env(parent = emptyenv())
+gsconfig$orignial.par <- par(no.readonly = TRUE)
```

Comment:
just so this doesn't mix us up later - the n and the i are mixed up here: original